tests/test_send.py

```diff
@@ -0,0 +1,113 @@
+import pytest
+
+from fastapi import FastAPI
+from fastapi.testclient import TestClient
+from pytest_mock import MockerFixture
+from unittest.mock import Mock
+
+from regtech_api_commons.models.auth import AuthenticatedUser
+from starlette.authentication import AuthCredentials
+
+
+@pytest.fixture
+def app_fixture(mocker: MockerFixture) -> FastAPI:
+    from api import app
+
+    return app
+
+
+@pytest.fixture
+def auth_mock(mocker: MockerFixture) -> Mock:
+    return mocker.patch(
+        "regtech_api_commons.oauth2.oauth2_backend.BearerTokenAuthBackend.authenticate"
+    )
+
+
+@pytest.fixture
+def user_no_profile_mock(auth_mock: Mock) -> Mock:
+    claims = {
+        "email": "test@cfpb.gov",
+    }
+    auth_mock.return_value = (
+        AuthCredentials(["authenticated"]),
+        AuthenticatedUser.from_claim(claims),
+    )
+    return auth_mock
+
+
+@pytest.fixture
+def full_user_mock(auth_mock: Mock) -> Mock:
+    claims = {
+        "name": "Test User",
+        "email": "test@cfpb.gov",
+    }
+    auth_mock.return_value = (
+        AuthCredentials(["authenticated"]),
+        AuthenticatedUser.from_claim(claims),
+    )
+    return auth_mock
+
+
+class TestEmailApiSend:
+
+    def test_send_no_profile(
+        self, mocker: MockerFixture, app_fixture: FastAPI, user_no_profile_mock: Mock
+    ):
+        email_json = {
+            "email": {
+                "subject": "Institution Profile Change",
+                "body": "lei: 1234567890ABCDEFGHIJ\ninstitution_name_1: Fintech 1\ntin_1: 12-3456789\nrssd_1: 1234567",
+                "from_addr": "test@cfpb.gov",
+                "sender": "<test@cfpb.gov>",
```

Comment:
Is it OK to send it with the wrapping `<>` when there's no friendly name? I would assume those wouldn't be there at all if `name` wasn't set.

Reply:
Oh I dunno, I can make the logic smarter here based on name being an empty string or not. (See below)

Reply:
From the specification (RFC 5322)
“ Normally, a mailbox is composed of two parts: (1) an optional display name that indicates the name of the recipient (which can be a person or a system) that could be displayed to the user of a mail application, and (2) an addr-spec address enclosed in angle brackets ("<" and ">"). There is an alternate simple form of a mailbox where the addr-spec address appears alone, without the recipient's name or the angle brackets. The Internet addr-spec address is described in section 3.4.1.”

so it sounds like brackets are fine but can be left on without the optional display name, or can removed.  I’m good either way